platform/use-platform/virtual-clusters/add-host-cluster.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Add existing host clusters
+sidebar_label: Add Existing Host Clusters
+sidebar_position: 3
+description: Add externally deployed host clusters to Platform with visibility-only or full management capabilities
+---
+
+import PageVariables from '@site/src/components/PageVariables';
+import InterpolatedCodeBlock from '@site/src/components/InterpolatedCodeBlock';
+
+You can register existing host clusters to an installed vCluster Platform running in another host cluster.  
+This enables centralized management and visibility of virtual clusters running across multiple host clusters from a single Platform deployment.
+
+### Register host cluster with CLI
+In the examples below:
+- `HOST_CLUSTER_NAME` — The unique identifier of the host cluster to be registered that is used by the vCluster Platform.  
+  It is recommended to use the same name as the actual Kubernetes host cluster.
+- `DISPLAY_NAME` — The display name shown in the Platform UI.
+- `HOST_CLUSTER_CONTEXT` — The Kubernetes context of the host cluster being registered.
+
+<PageVariables HOST_CLUSTER_NAME="cluster-x" HOST_CLUSTER_CONTEXT="cluster-x-context"  DISPLAY_NAME="dev-cluster" />
+
+First, switch to the kubernetes context of the host cluster to be registered:
+<InterpolatedCodeBlock
+  code={`kubectl config use-context [[GLOBAL:HOST_CLUSTER_CONTEXT]]`}
+  language="bash"
+/>
+
+Then, install the platform agent through the CLI:
+
+<InterpolatedCodeBlock
+  code={`vcluster platform add cluster [[GLOBAL:HOST_CLUSTER_NAME]] \\
+    --display-name "[[GLOBAL:DISPLAY_NAME]]"`}
+  language="bash"
+/>
+
+This will deploy the vCluster Platform agent in the host cluster. By default, `vcluster-platform` namespace will be created if it doesn't exist already and the platform agent pods will be running in this namespace.
+You can find more CLI flags information at [CLI documentation](../../../../vcluster/cli/vcluster_platform_add_cluster).
+
+After the command is finished, you can verify the host is registered in the platform by running
+
+<InterpolatedCodeBlock
+  code={`vcluster platform list clusters`}
+  language="bash"
+/>
+The expected output will be:
+```bash
+         CLUSTER      |  AGE
+  --------------------+--------
+    cluster-a         | 21m
+    cluster-x         | 7s
+```
+
+### Upgrade platform agent
```

Comment:
This needs more context why would one need to upgrade the agent after adding an existing host cluster.